wss/1.0/modules/ROOT/partials/transformation-note.adoc

```diff
@@ -0,0 +1,4 @@
+[NOTE]
+--
+Transform operations don't take into account the cryptographic signatures and can end up modifying them.
```

Comment:
If this note applies to transform operations in general, you can remove "the" in front of "cryptographic signatures".